src/controllers/controller.bubble.js

```diff
@@ -79,6 +79,18 @@ module.exports = DatasetController.extend({
 		return parsed;
 	},
 
+	getMaxOverflow: function() {
```

Comment:
I might make this private for now in case we decide to change the API. E.g. if we want to handle start and end separately

Reply:
done